package/MDAnalysis/lib/formats/include/correl.h

```diff
@@ -0,0 +1,188 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode:nil; -*- */
+/* vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 */
+/*
+  MDAnalysis --- http://mdanalysis.googlecode.com
+  Copyright (c) 2006-2014 Naveen Michaud-Agrawal,
+                Elizabeth J. Denning, Oliver Beckstein,
+                and contributors (see AUTHORS for the full list)
+  Released under the GNU Public Licence, v2 or any higher version
+
+  Please cite your use of MDAnalysis in published work:
+
+      N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and
+      O. Beckstein. MDAnalysis: A Toolkit for the Analysis of
+      Molecular Dynamics Simulations. J. Comput. Chem. 32 (2011), 2319--2327,
+      in press.
```

Comment:
See also mdanalysis/MDAnalysis.github.io#27 – you could open a new issue for a new comment header on all files and we can discuss there what we want to put into the new header.